[2023-01-13T12:00:01.222+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-13T00:00:00+00:00 [queued]>
[2023-01-13T12:00:01.228+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-13T00:00:00+00:00 [queued]>
[2023-01-13T12:00:01.228+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-13T12:00:01.228+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-13T12:00:01.228+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-13T12:00:01.236+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_avaible> on 2023-01-13 00:00:00+00:00
[2023-01-13T12:00:01.240+0000] {standard_task_runner.py:55} INFO - Started process 814 to run task
[2023-01-13T12:00:01.242+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_avaible', 'scheduled__2023-01-13T00:00:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpa8q3zzl_']
[2023-01-13T12:00:01.243+0000] {standard_task_runner.py:83} INFO - Job 125: Subtask is_api_avaible
[2023-01-13T12:00:01.282+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_avaible scheduled__2023-01-13T00:00:00+00:00 [running]> on host 274e1da73cdf
[2023-01-13T12:00:01.324+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_avaible
AIRFLOW_CTX_EXECUTION_DATE=2023-01-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-13T00:00:00+00:00
[2023-01-13T12:00:01.324+0000] {http.py:120} INFO - Poking: api/
[2023-01-13T12:00:01.330+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-01-13T12:00:01.330+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-01-13T12:00:02.003+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-01-13T12:00:02.021+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_avaible, execution_date=20230113T000000, start_date=20230113T120001, end_date=20230113T120002
[2023-01-13T12:00:02.057+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-13T12:00:02.102+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
